### ISSUE TITLE
Raspberry Pi 4 build instructions 

### ISSUE THREAD

**Issue by kevinm54** [groupgets/GetThermal#42]
There have been several issues dealing with bugs related to the Raspberry Pi 4 including #29 and #23 . The wiki instructions have not been updated to reflect this. There seem to be specific steps that need to be taken such as building a non-master branch or applying a patch. These instructions should be incorporated into the wiki instructions.

**Comment by kekiefer** [comment]
Since you're working through this now, could you summarize what needs to be updated?

**Comment by kevinm54** [comment]
I will if I can figure it out. I haven't gotten it to work yet.

**Comment by bobbasham** [comment]
I am also having problems with setting up Raspberry Pi 4 (Raspberry Pi OS)/Purethermal2/GetThermal
Has there been any success with this?  I have tried every method I can find available.

**Comment by Werterror** [comment]
Any updates regarding those issues?

**Comment by girgine** [comment]
used the following for qt5 related error and kept the rest same, its currently working.. 
sudo apt-mark manual qtbase5-dev qtchooser qt5-qmake qtbase5-dev-tools

article is here: https://forum.qt.io/topic/132190/issue-with-sudo-apt-get-install-qt5-default/2

**Comment by GeorgeTasop** [comment]
As @girgine mentioned above and according to the provided [link](https://forum.qt.io/topic/132190/issue-with-sudo-apt-get-install-qt5-default/2) the correct command for Raspberry Pi 4 with Rasbpian bullseye, in [step 2](https://github.com/groupgets/GetThermal/wiki/Building-for-Raspberry-Pi#2-install-development-packages) of the Wiki should be:

```
sudo apt-get install qtbase5-dev qtchooser qt5-qmake \
    qtbase5-dev-tools qtmultimedia5-dev qtdeclarative5-dev \
    qml-module-qtquick-controls2 qml-module-qtmultimedia \
    qml-module-qtquick-layouts qml-module-qtquick-window2 \
    qml-module-qtquick-templates2 qml-module-qtgraphicaleffects \
    libusb-1.0-0-dev cmake git
```
Otherwise, I followed the process as mentioned in the Wiki and it worded perfectly.

Maybe it can be updated with a note that this command is just for Rasbpian bullseye @kekiefer ? Thank you!
